.mergify.yml

```diff
@@ -11,6 +11,15 @@ pull_request_rules:
         # wait for OBS package build as well which is triggered independantly
         # See https://progress.opensuse.org/issues/55346 for details
         - "check-success=OBS Package Build"
+        # Multibuild results will not report the "pending" status, so we
+        # need to require them explicitly, see
+        # https://openbuildservice.org/help/manuals/obs-user-guide/cha.obs.scm_ci_workflow_integration.html#sec.obs.obs_scm_ci_workflow_integration.setup.status_reporting
```

Comment:
Is there a feature request for this? Because hard coding here shows how this is problematic also for other use cases

Reply:
I don't know of a feature request.

Reply:
Can you create one? Like the bug report you created for the open build service, just a different GitHub issue template :)

By the way, as alternative to spelling out each check for mergify we can also look for a count of checks that must be successful. This way we wouldn't need to hardcore everything. If you don't like that please use at least a generic pattern for the Leap version so that we don't need to change all these numbers with every point-release of Leap

Reply:
I changed the check for Leap to a regex now

Reply:
Created an issue: https://github.com/openSUSE/open-build-service/issues/12121